src/sql/workbench/browser/designer/designer.ts

```diff
@@ -941,6 +1031,54 @@ export class Designer extends Disposable implements IThemable {
 		return component;
 	}
 
+	private addDesignerTaskbar(container: HTMLElement, tableProperties: DesignerTableProperties): void {
+		if (tableProperties.canAddRows || tableProperties.canMoveRows) {
+			let taskbarContainer = container.appendChild(DOM.$('.full-row')).appendChild(DOM.$('.add-row-button-container'));
+			this._taskbar = new Taskbar(taskbarContainer);
+			this._actions = [];
```

Comment:
I've mentioned the problem with this._taskbar elsewhere. not sure by disposing the taskbar whether the actions inside it will be disposed too, if so, you don't need the this._actions global variable.